table/table.go

```diff
@@ -291,6 +291,12 @@ func (m Model) Cursor() int {
 	return m.cursor
 }
 
+// SetCursor sets the cursor position in the table.
+func (m *Model) SetCursor(n int) {
```

Comment:
I wonder if `MoveTo` is a better name, to match `MoveUp` and `MoveDown`.

Reply:
I used SetCursor since it matches with `Cursor` and the others `Set*` methods, and it literally sets the `cursor` attribute :-) But it's up to you, i don't have a preference on this, let me know if you want me to amend the commit.